tests/test_tables.py

```diff
@@ -1158,6 +1158,11 @@ def test_remove_single(table):
     z      | 1     | 10
     """)
 
+def test_remove_none(table):
+	num_rows_before = table.num_rows
```

Comment:
Seems like your tabs setting is set to 8 spaces versus the 4 spaces in the rest of this file. Can you please recheck this and make sure that the tabs look uniform?

Reply:
Thanks for the feedback on the tab spacing! I've made the changes to ensure consistency. Could you please take another look and confirm if it looks correct on your end?

Reply:
Unfortunately, it doesn’t look like your new commit did anything :( the diff is empty and the indent still looks like 8 spaces (check the commits tab here on the PR). Can you please try again?

Reply:
You were right, I did check and the commit had not changed the spacing. Please verify now. Thank you

Reply:
Looks good to me, thank you!